src/graphql/operations/users.ts

```diff
@@ -27,19 +29,52 @@ export default async function (parent, args) {
   const query = `
     SELECT
       u.*,
-      SUM(l.vote_count) as votesCount,
-      SUM(l.proposal_count) as proposalsCount,
+      COALESCE(SUM(l.vote_count), 0)  as votesCount,
+      COALESCE(SUM(l.proposal_count), 0) as proposalsCount,
       MAX(l.last_vote) as lastVote
     FROM users u
-    INNER JOIN leaderboard l ON l.user = u.id
+    LEFT JOIN leaderboard l ON l.user = u.id
     WHERE 1=1 ${queryStr}
     GROUP BY u.id
     ORDER BY ${orderBy} ${orderDirection} LIMIT ?, ?
   `;
   params.push(skip, first);
   try {
     const users = await db.queryAsync(query, params);
-    return users.map(user => formatUser(user));
+    ids.forEach(element => {
+      if (!users.find((u: any) => u.id === element)) {
+        users.push({ id: element });
+      }
+    });
+    if (!users.length) return [];
+
+    const usersWithOutCreated = users
+      .filter((u: any) => !u.created)
+      .map((u: any) => u.id);
+    if (usersWithOutCreated.length) {
+      const counts = await db.queryAsync(
+        `
+        SELECT
+          user,
+          COALESCE(SUM(vote_count), 0) as votesCount,
+          COALESCE(SUM(proposal_count) ,0) as proposalsCount,
+          MAX(last_vote) as lastVote
+        FROM leaderboard
+        WHERE user IN (?)
+        GROUP BY user
+      `,
+        [usersWithOutCreated]
+      );
+      counts.forEach((count: any) => {
+        const user = users.find((u: any) => u.id === count.user);
+        if (user) {
+          user.votesCount = count.votesCount;
+          user.proposalsCount = count.proposalsCount;
+          user.lastVote = count.lastVote;
+        }
```

Comment:
```suggestion
        user.votesCount = count.votesCount;
        user.proposalsCount = count.proposalsCount;
        user.lastVote = count.lastVote;
        
```

Reply:
The case where `user` does not exist will never happen, as we get the ID from this object